index.html

```diff
@@ -0,0 +1,190 @@
+                      <h2 class="date">2022.06.25(Fri) - 27(Sun)</h2>
+                      <p class="address">@Eko Hotel and Suites, Lagos, Nigeria.</p>
+                    </div>
+                  </div>
+                </div>
+              </div>
+        </section>
+        <section id="program" class="main-program text-white py-5">
+            <div class="container py-md-5 py-3">
+                <div class="row">
+                    <h3 class="section-title light">Main Program</h3>
+                  </div>
+                  <div class="row py-md-3">
+                    <div class="main-program-items">
+                      <article class="main-program-item">
+                        <img class=" mx-md-auto" src="./img/icons/talk.png" alt="speaker-icon">
+                        <h4 class="title">Lectures</h4>
+                        <p class="text-md-center">Listen to the speakers from various countries about the messages of sharing and opening.</p>
+                      </article>
+                      <article class="main-program-item">
+                        <img class=" mx-md-auto" src="./img/icons/exhibition.png" alt="speaker-icon">
+                        <h4 class="title">Exhibition</h4>
+                        <p class="text-md-center">Appreciate various creations applying CC license of artists, organized from Art Center Nabi.</p>
+                      </article>
+                      <article class="main-program-item">
+                        <img class=" mx-md-auto" src="./img/icons/live-chat.png" alt="speaker-icon">
+                        <h4 class="title">Forum</h4>
+                        <p class="text-md-center">Have the time your thoughts and opinions width experts for each topic.</p>
+                      </article>
+                      <article class="main-program-item">
+                        <img class=" mx-md-auto" src="./img/icons/workshop.png" alt="speaker-icon">
+                        <h4 class="title">Workshop</h4>
+                        <p class="text-md-center">Try creating your own work using open source license rather than just watching at it.</p>
+                      </article>
+                      <article class="main-program-item">
+                        <img class=" mx-md-auto" src="./img/icons/network.png" alt="speaker-icon">
+                        <h4 class="title">Network</h4>
+                        <p class="text-md-center">Listen to Get opportunities to network with CC affiliates around the world, also after the summit.</p>
+                      </article>
+                    </div>
+                  </div>
+                  <div class="row pt-md-5 pt-3">
+                    <div class="col d-flex justify-content-center pt-3">
+                      <a href="./about.html" class=" text-center">See full program</a>
+                    </div>
+                  </div>
+            </div>
+        </section>
```

Comment:
Great use of semantics 😎 The proper use of tags is key to having better HTML! Well done using `section`, `article`, and `footer` tags 💪